services/ui-src/src/components/pages/Dashboard/DashboardTable.tsx

```diff
@@ -43,7 +46,10 @@ export const DashboardTable = ({
         )}
         {/* Report Name */}
         <Td sx={sxOverride.programNameText}>
-          {report.programName ?? report.submissionName}
+          {report.state} {report.submissionName ?? report.programName} {"{"}
+          {year} {" - Period"}{" "}
+          {calculatePeriod(convertDateUtcToEt(report.createdAt))}
+          {"}"}
```

Comment:
since this is a little complex you could consider moving this out to a function within this file and just calling it here like
```
{getReportName(report)}
```
or something. Then you can also generate the year variable within that function. It doesn't really matter in the end, just makes reading this component a little cleaner!

Reply:
good point! I will create a function for this! 😄